Sources/StreamChat/Database/DatabaseContainer_Tests.swift

```diff
@@ -38,6 +38,7 @@ class DatabaseContainer_Tests: XCTestCase {
                 userDTO.userCreatedAt = .init()
                 userDTO.userUpdatedAt = .init()
                 userDTO.userRoleRaw = "user"
+                userDTO.teams = []
```

Comment:
Would be nice to dedicate some time and revisit the API of creating DTOs. Ideally, there should be a single way of creating an object with all non-optional attributes as input params (`init` 😄?) leaving a developer 0 chance to instantiate the instance with an unassigned mandatory field.